src/main/java/com/eviive/personalapi/controller/ProjectController.java

```diff
@@ -177,7 +193,11 @@ public ResponseEntity<Void> sort(@RequestBody final List<SortUpdateDTO> sorts) {
         summary = "Delete a project",
         responses = {
             @ApiResponse(responseCode = "204", description = "No Content"),
-            @ApiResponse(responseCode = "400", description = "Bad Request")
+            @ApiResponse(
+                responseCode = "404",
+                description = "Bad Request",
+                content = @Content(schema = @Schema(implementation = ProblemDetail.class))
+            )
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'annotation array initialization' child has incorrect indentation level 12, expected level should be 16.